nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/service/StandardConfigurationContext.java

```diff
@@ -78,7 +78,9 @@ public PropertyValue getProperty(final PropertyDescriptor property) {
         // We need to get the 'canonical representation' of the property descriptor from the component itself,
         // since the supplied PropertyDescriptor may not have the proper default value.
         final PropertyDescriptor resolvedDescriptor = component.getPropertyDescriptor(property.getName());
-        final String resolvedValue = (configuredValue == null) ? resolvedDescriptor.getDefaultValue() : configuredValue;
```

Comment:
I don't think this code should be changing at all. We should not use the default value of the given Property Descriptor, but rather use the default value of the one provided by the Controller Service. It appears that the Controller Service is not actually providing a Property Descriptor for this property, which means that it should return `null`. If the Controller Service wants to make use of that property, it needs to expose that property in its `getSupportedPropertyDescriptors` or `getSupportedDynamicPropertyDescriptor` method